Description: Script for bootstrapping upstreams build system
 When upstreams build system is outdated this script can be run by running
 quilt push -f 010_rebootstrap.diff
 sh ./bootstrap
 quilt refresh (maybe be a little smarter for missing files)
Forwarded: not-needed
Author: Sam Hocevar (Debian packages) <sam+deb@zoy.org>
Author: Paul Gevers <paul@climbing.nl>
Index: lesstif2-0.95.2/bootstrap
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ lesstif2-0.95.2/bootstrap	2009-07-29 13:11:38.000000000 +0200
@@ -0,0 +1,26 @@
+#! /bin/sh
+
+for x in aclocal.m4 configure config.guess config.log config.sub config.cache config.h.in config.h compile libtool.m4 ltoptions.m4 ltsugar.m4 ltversion.m4 ltmain.sh libtool ltconfig missing mkinstalldirs depcomp install-sh; do rm -f $x test/$x; done
+rm -Rf autom4te.cache test/autom4te.cache
+
+# Explain what we are doing from now
+set -x
+
+libtoolize --copy --force --automake
+aclocal-1.9 -I .
+autoconf
+autoheader
+automake-1.9 --foreign --add-missing --copy --force-missing --ignore-deps
+
+# Remove cruft that we no longer want
+rm -Rf autom4te.cache
+cd test
+
+libtoolize --copy --force --automake
+aclocal-1.9 -I ..
+autoconf
+autoheader
+automake-1.9 --foreign --add-missing --copy --force-missing --ignore-deps
+
+# Remove cruft that we no longer want
+rm -Rf autom4te.cache
